string-manipulation-1/src/reverse-word.js

```diff
@@ -0,0 +1,8 @@
+/* exported reverseWord */
+function reverseWord(word) {
+  var reverseArr = [];
+  for (var i = word.length - 1; i >= 0; i--) {
+    reverseArr.push(word[i]);
+  }
+  return reverseArr.join('');
```

Comment:
Instead of using an array you need to join, you should just concatenate together the characters into a string.